lib/ex_doc/refs.ex

```diff
@@ -100,17 +100,14 @@ defmodule ExDoc.Refs do
   defp fetch_entries(module, result) do
     case result do
       {:docs_v1, _, _, _, module_doc, _, docs} ->
-        module_visibility = visibility(module_doc)
+        module_visibility = visibility({module, module_doc})
```

Comment:
I don't see why we pass the module, we never use it. :)

Reply:
True. fixed